release/api/v1alpha1/bundle_types.go

```diff
@@ -242,10 +242,11 @@ type FluxBundle struct {
 }
 
 type PackageBundle struct {
-	Version        string `json:"version,omitempty"`
-	Controller     Image  `json:"packageController"`
-	TokenRefresher Image  `json:"tokenRefresher"`
-	HelmChart      Image  `json:"helmChart,omitempty"`
+	Version                   string `json:"version,omitempty"`
+	Controller                Image  `json:"packageController"`
+	TokenRefresher            Image  `json:"tokenRefresher"`
+	CredentialProviderPackage Image  `json:"credentialProviderPackage"`
```

Comment:
What are your thoughts about this? 
```suggestion
	CredentialProvider Image  `json:"credentialProvider"`
```

And similar change everywhere else

Reply:
The package suffix may look weird, but I think we should better match this name in packageBundle:
```
apiVersion: packages.eks.amazonaws.com/v1alpha1
kind: PackageBundle
metadata:
  annotations:
    eksa.aws.com/excludes: LnNwZWMucGFja2FnZXNbXS5zb3VyY2UucmVnaXN0cnkKLnNwZWMucGFja2FnZXNbXS5zb3VyY2UucmVwb3NpdG9yeQo=
    eksa.aws.com/signature: MEUCIQDmbyJz/gOWbvtMN2bmYSrZ/p2GATd5MW5oOxkWokTctAIgAc5VkzVNZmzkP6MjX9bwA4Fx9jdP55qZ7NP1l7CSlQg=
  name: v1-24-2068
  namespace: eksa-packages
spec:
  minControllerVersion: v0.3.2
  packages:
  - name: eks-anywhere-packages
    source:
      repository: eks-anywhere-packages
      versions:
      - dependencies:
        - eks-anywhere-packages-crds
        digest: sha256:ccd46de5b5aa7e3d726abbcc93b599f634c14bdfbb141016e0832d19d906a3a1
        images:
        - digest: sha256:99aee59e6243e8b35e763d08f511253a1db9f1ab2a8254ccc6630afcd0da2379
          repository: eks-anywhere-packages
        - digest: sha256:fe16d6dfb4db71a48d4116e8de21103e20dca1a22702f20f0dab7277ddc4d8c7
          repository: ecr-token-refresher
        name: 0.0.0-b2012b657faa6659030863b3d5d8f86607aaec65-release-0.16-helm
        schema: H4sIABcgp2QAA6VUwXKCMBC98xVMpseKd2+91Qsy1kM7HaezDatGQ0KTgKWO/96ASAMFp9oDQ/L2vZfdLMvB831yx2Iy8cnGmFRPxuMU6A7WqAPc6QAS+JIC9jqgMhlbZASi2G9Q4ajhabrBBIKtloLcl4aGGY6lZS+/5hRpRZHvW6TmhKVKpqgMs5yJf7CIxagURknOUTVYv7rCexwqHEXeAhwLUAqKxqEKMYOJ7vCHD62jCj8yprC8ytdWxMYEJNjhWzQHniFpocuOKcQxM8w2gEduZSvgGjvUgdLdDH7jTlXaKCbWpEM4DmR9i5U3tPtZn1end314Wdpn8Z/2Py4W0Vs0nz2/DH0F3ZSduiv10+3ycHaltvcOtMwUxTmumeX3Xobrc1ZRnmmDKmx3/6IkVSwHc/VJLLGzHWWcR5IzeknWdAtFlrSmhUxXoTSRQo3CnS/ywPdQaBcJMbd/hHq/bGUS4woybq4toJZNyzr+rPXK5+h9A3zb6LRHBQAA
  - name: eks-anywhere-packages-crds
    source:
      repository: eks-anywhere-packages-crds
      versions:
      - dependencies:
        - eks-anywhere-packages-migrations
        digest: sha256:e6b17ab8eae80616746a0ecfb77daeb4e1866e71a4a8d0b2a0109ec44ba92eac
        images:
        - digest: sha256:99aee59e6243e8b35e763d08f511253a1db9f1ab2a8254ccc6630afcd0da2379
          repository: eks-anywhere-packages
        - digest: sha256:fe16d6dfb4db71a48d4116e8de21103e20dca1a22702f20f0dab7277ddc4d8c7
          repository: ecr-token-refresher
        name: 0.0.0-b2012b657faa6659030863b3d5d8f86607aaec65
        schema: H4sIABUgp2QAA6VUwXKCMBC98xVMpseKd2+91Qsy1kM7HaezDatGQ0KTgKWO/96ASAMFp9oDQ/L2vZfdLMvB831yx2Iy8cnGmFRPxuMU6A7WqAPc6QAS+JIC9jqgMhlbZASi2G9Q4ajhabrBBIKtloLcl4aGGY6lZS+/5hRpRZHvW6TmhKVKpqgMs5yJf7CIxagURknOUTVYv7rCexwqHEXeAhwLUAqKxqEKMYOJ7vCHD62jCj8yprC8ytdWxMYEJNjhWzQHniFpocuOKcQxM8w2gEduZSvgGjvUgdLdDH7jTlXaKCbWpEM4DmR9i5U3tPtZn1end314Wdpn8Z/2Py4W0Vs0nz2/DH0F3ZSduiv10+3ycHaltvcOtMwUxTmumeX3Xobrc1ZRnmmDKmx3/6IkVSwHc/VJLLGzHWWcR5IzeknWdAtFlrSmhUxXoTSRQo3CnS/ywPdQaBcJMbd/hHq/bGUS4woybq4toJZNyzr+rPXK5+h9A3zb6LRHBQAA
  - name: eks-anywhere-packages-migrations
    source:
      repository: eks-anywhere-packages-migrations
      versions:
      - digest: sha256:9900ff00be007e3e10cd064222cfa514ae306d717dc43ababdd8ecdb14b97a88
        images:
        - digest: sha256:99aee59e6243e8b35e763d08f511253a1db9f1ab2a8254ccc6630afcd0da2379
          repository: eks-anywhere-packages
        - digest: sha256:fe16d6dfb4db71a48d4116e8de21103e20dca1a22702f20f0dab7277ddc4d8c7
          repository: ecr-token-refresher
        name: 0.0.0-b2012b657faa6659030863b3d5d8f86607aaec65
        schema: H4sIABMgp2QAA6VUwXKCMBC98xVMpseKd2+91Qsy1kM7HaezDatGQ0KTgKWO/96ASAMFp9oDQ/L2vZfdLMvB831yx2Iy8cnGmFRPxuMU6A7WqAPc6QAS+JIC9jqgMhlbZASi2G9Q4ajhabrBBIKtloLcl4aGGY6lZS+/5hRpRZHvW6TmhKVKpqgMs5yJf7CIxagURknOUTVYv7rCexwqHEXeAhwLUAqKxqEKMYOJ7vCHD62jCj8yprC8ytdWxMYEJNjhWzQHniFpocuOKcQxM8w2gEduZSvgGjvUgdLdDH7jTlXaKCbWpEM4DmR9i5U3tPtZn1end314Wdpn8Z/2Py4W0Vs0nz2/DH0F3ZSduiv10+3ycHaltvcOtMwUxTmumeX3Xobrc1ZRnmmDKmx3/6IkVSwHc/VJLLGzHWWcR5IzeknWdAtFlrSmhUxXoTSRQo3CnS/ywPdQaBcJMbd/hHq/bGUS4woybq4toJZNyzr+rPXK5+h9A3zb6LRHBQAA
  - name: credential-provider-package
    source:
      repository: credential-provider-package
      versions:
      - digest: sha256:9007f8ee20aba2029aad91b0b532a07970e080186a6f720ad365cfd87114793b
        images:
        - digest: sha256:a7983e32cc9c32e1b22bba34ec4034e1487f9a8c8f5040ee60f92c29b3cb4bc9
          repository: credential-provider-package
        name: 0.0.0-b2012b657faa6659030863b3d5d8f86607aaec65
        schema: H4sIABkgp2QAA7WVzW7CMAzH7zwF6nbsxp0bYpddNjSeIKRuCWrjyHHREOLdZ9oNlfVzo/RQtU7zsx3/7R4nU7mCR6+3kKlgPg22zG4+m+082qfS+oyUzMrHhyAsN/DBwflr3OxA84/VETogNuBl7VjYCruKY2MNH66sTZzL4ims7HYuNVqxQdsLCK9XWwK6rEcQqzzlpZLsXnJq9lHz5ZmMTYLaR6ew7iFTrLevmUpaQrgCKyJ1GMaVzGKTwrjBetAE/KayW7iT5rdqRb+P/ezIO6Xr3lq9VClxnqZWEO97IDLRPynmXJvRdWWk3jxucZyku0LphHob3cQlcOgNI43MZZXcV0VF5VZyKutCtvVStPVVFXK7gCxGsIZUVIL0vwHnMFpYi1yMn440ehhyDDnJlF2iZfisq28Qh8BjTrpB2IO2+zFi8EB7o2GhNea2n/HH3lQdR90bZkO4lz0iQsUDJucGMQVlh0Ht3YdxWe8PSIxQOn7RXT3AkhH1yfd3F07K+2nyBbU3NaSFCAAA
```